specification/dimensions/provider.md

```diff
@@ -0,0 +1,32 @@
+# Provider
+
+A Provider is an entity that made the infrastructure and/or services available for purchase. It is commonly used for cost analysis and reporting scenarios.
```

Comment:
nit: Is "cost analysis and reporting scenarios" helpful? We could arguably say that about every column. I feel like we could leave that sentence out and not lose anything. Is it needed or should we give more specific scenario examples?

Reply:
I agree.
in this chapter, in this iteration I just followed the concept used for Service Name, Resource ID... (only left out the allocation)